.github/workflows/test.yaml

```diff
@@ -251,6 +251,19 @@ jobs:
           # Stop local network services
           pkill hc-run-local
 
+      - name: Smoke test - kitsune
+        run: |
+          set -x
+
+          # Start local bootstrap and signal server
+          nix develop .#ci -c bash -c "kitsune2-bootstrap-srv --listen 127.0.0.1:30000 &"
+
+          # Run the scenario
+          RUST_LOG=warn cargo run -p kitsune_scenario -- --bootstrap-server-url http://127.0.0.1:30000 --signal-server-url ws://127.0.0.1:30000 --duration 40 --agents 2
+
+          # Stop servers
+          pkill kitsune2-bootst
```

Comment:
```suggestion
          pkill kitsune2-bootstrap-srv
```

Reply:
I did this because it's what pkill completes it to when hitting <TAB>, and the `hc-run-local-services` higher up in the file is abbreviated too.

Reply:
https://github.com/holochain/wind-tunnel/pull/147/commits/07b4f7afe135565ac284c1b6e4bf974add2059de